Simplenote/SimplenoteConstants.swift

```diff
@@ -22,18 +22,19 @@ class SimplenoteConstants: NSObject {
     static let simplenoteInterlinkHost = "note"
     static let simplenoteInterlinkMaxTitleLength = 150
 
-    static let currentEngineBaseURL = "https://app.simplenote.com" as NSString
+    static let googleAppEngineBaseURL = "https://app.simplenote.com" as NSString
+    static let googleAppEngineHost = "app.simplenote.com"
```

Comment:
I've renamed `currentEngineBaseURL` > `googleAppEngineBaseURL`, because, of course it's current 😆